packages/shared/routes/onboarding/views/ledger-setup/views/InstallationGuideView.svelte

```diff
@@ -26,22 +26,15 @@
     <div slot="leftpane__content">
         <Text type="h2" classes="mb-5">{localize('views.ledgerInstallationGuide.title')}</Text>
         <Text type="p" secondary classes="mb-5">
-            {localize('views.ledgerInstallationGuide.body1', { values: { legacy: LedgerAppName.IOTALegacy } })}
+            {localize('views.ledgerInstallationGuide.body1')}
         </Text>
         <Text type="p" secondary classes="mb-5">{localize('views.ledgerInstallationGuide.body2')}</Text>
         <div class="flex flex-row flex-nowrap items-center justify-center space-x-4 text-center mt-28">
             <div class="flex flex-col flex-wrap space-y-2">
                 <div class="bg-blue-400 rounded-2xl w-20 h-20 flex justify-center items-center">
-                    <Icon icon="ledger-app-legacy" width="32" height="32" classes="text-white" />
+                    <Icon icon="shimmer" width="32" height="32" classes="text-white" />
```

Comment:
We should double check if we only want to show this view during Ledger onboarding for only Shimmer and not IOTA. I assumed we will show for both, but since only the Shimmer app is new maybe it makes sense to leave it?